### PR TITLE
Adds Cassini Vims driver for use within ISIS

### PR DESCRIPTION
**Pull request by jessemapel**
Re-do of #432

### CODE REVIEW

**Comment by acpaquette** [comment]
Passing tests

Screen Shot 2023-03-20 at 1 42 38 PM

File: ale/drivers/co_drivers.py [hunk 1]
```diff
@@ -293,6 +308,204 @@ def frame_chain(self):
...
+        -------
+        : str
+          instrument id
+        """
+        return vims_id_lookup[super().instrument_id + "_" + self.vims_channel]
+
+    @property
+    def sensor_name(self):
+        """
+        ISIS doesn't propergate this to the ingested cube label, so hard-code it.
```
Comment (AustinSanders) [inline ale/drivers/co_drivers.py:338]: propergate -> propagate

File: ale/drivers/co_drivers.py [hunk 2]
```diff
@@ -293,6 +308,204 @@ def frame_chain(self):
...
+          instrument id
+        """
+
+        image_type = self.label['IsisCube']["Instrument"]["Channel"]
+        return vims_id_lookup[super().instrument_id + "_" + image_type]
+
+    @property
+    def sensor_name(self):
+        """
+        ISIS doesn't propergate this to the ingested cube label, so hard-code it.
```
Comment (AustinSanders) [inline ale/drivers/co_drivers.py:467]: propergate -> propagate

**Review by AustinSanders — COMMENTED** [review]
Quick spelling correction and then good to merge.  Sorry to be annoying :)

**Review by AustinSanders — APPROVED** [review]

**Pull request merged**